### PR TITLE
Weekend changes

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: TeamCode/src/main/java/intothedeep/CraneCapabilities.java [hunk 1]
```diff
@@ -8,32 +8,47 @@ public class CraneCapabilities {
...
         this.craneStabilizer = new PIDController(0.01, 0, 0);
     }
 
-    public void runCrane(double power) {
-        this.c.liftLeft.setPower(power);
-        this.c.liftRight.setPower(power);
-        this.craneTarget = this.c.liftLeft.motor.getCurrentPosition();
+    public void runCrane(double speed) {
+        this.c.liftLeft.motor.setVelocity(speed*c.liftLeft.ticksPerRevolution);
+        this.c.liftRight.setPower(-speed*c.liftRight.ticksPerRevolution);
```
Comment (lllincoln) [inline TeamCode/src/main/java/intothedeep/CraneCapabilities.java]: setPower

File: TeamCode/src/main/java/intothedeep/auto/generatedAutos/SquareTestAuto.java [hunk 2]
```diff
@@ -9,14 +9,14 @@
 @Autonomous(name = "SquareTestAuto")
 public class SquareTestAuto extends EasyAuto {
 	public SquareTestAuto() {
-		super(Alliance.RED);
+		super(Alliance.NULL);
```
Comment (lllincoln) [inline TeamCode/src/main/java/intothedeep/auto/generatedAutos/SquareTestAuto.java]: make this just null

File: TeamCode/src/main/java/t10/utils/Alliance.java [hunk 3]
```diff
@@ -2,5 +2,6 @@
 
 public enum Alliance {
     RED,
-    BLUE
+    BLUE,
+    NULL
```
Comment (lllincoln) [inline TeamCode/src/main/java/t10/utils/Alliance.java]: remove

**Review by lllincoln — CHANGES_REQUESTED** [review]

File: TeamCode/src/main/java/intothedeep/auto/EasyAuto.java [hunk 4]
```diff
@@ -33,9 +34,9 @@ public EasyAuto(Alliance alliance, double startingTile) {
 
     @Override
     public void initialize() {
-        AbstractRobotConfiguration c = new SnowballConfig(this.hardwareMap);
-        this.driver = c.createMecanumDriver();
-        this.odometry = c.createOdometry();
+        this.config = new SnowballConfig(this.hardwareMap);
```
Comment (AncientBison) [inline TeamCode/src/main/java/intothedeep/auto/EasyAuto.java:37]: changed this to be `this.config` from just `config` for code style

**Review by lllincoln — APPROVED** [review]
good until we test

**Review by AncientBison — APPROVED** [review]
👍 👍 👍

**Pull request merged**